index.html

```diff
@@ -274,27 +383,27 @@ <h2>
       <form action="https://formspree.io/f/xeqnbeng" method="POST">
         <section class="contact">
           <div class="contact-text">
-            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in</h2>
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch</h2>
           </div>
-            <div><input type="text" name="name" id="fullname" placeholder="Full name" maxlength="30" required></div>
-            
-            <div><hr><input type="email" name="email" id="email" placeholder="Email address" required></div>
-            
-            <div class="div-form"><hr>
-              <textarea name="comments" 
-              id="text-area" 
-              cols="30"
-              rows="7"
-              maxlength="500"
-              placeholder="Enter text here"
-              required></textarea>
-            </div>
-            
-            <div>
-              <hr>
-              <button type="submit" id="contact-button" class="project-btn1">Get in touch</button>
-            </div>
-          <hr>
+          <div>
+              <div><input type="text" name="name" id="fullname" placeholder="First name" maxlength="30" required></div>
+              
+              <div>
+                <hr><input type="email" name="email" id="email" placeholder="Email address" required>
+              </div>
+              
+              <div class="div-form">
+                <hr>
+                <textarea name="comments" id="text-area" cols="30" rows="7" maxlength="500" placeholder="Enter text here" required></textarea>
+              </div>
+              
+              <div>
+                <button type="submit" id="contact-button" class="project-btn1">Get in touch</button>
+              </div>
+            <!-- <hr class="hr"> </hr> -->
+          </div>
+          <div><input type="text" name="name" id="full-name" placeholder="Last name" class="fullname" maxlength="30" required><hr></div>
+          
         </section>
       </form>
       
```

Comment:
- [ ] Kindly align the social links to match the Figma design as much as possible 🙏🏻🙏🏻✅✅
- Figma design
![image](https://user-images.githubusercontent.com/58553711/177675347-04b8a092-612f-490d-8b19-1103c261ee68.png)

- Your design
![image](https://user-images.githubusercontent.com/58553711/177675481-6494f6a3-5cc8-411e-9314-f1c7bf57a98f.png)